tests/rule_based_profiler/bob_user_workflow_verbose_profiler_config.yml

```diff
@@ -0,0 +1,231 @@
+    class_name: Rule
+    module_name: great_expectations.profiler.rule.rule
+    domain_builder:
+      # Build list of domains from all numeric column domains by checking column storage type using the configured execution engine
+      class_name: SimpleSemanticTypeColumnDomainBuilder
+      module_name: great_expectations.profiler.domain_builder.simple_semantic_type_domain_builder
+      semantic_types:
+        - numeric
+    parameter_builders:
+      # Note: my_column_mean and my_column_min are added as parameters but unused in expectation_configuration_builders - this is OK, we should still accept this as a valid Profiler configuration.
+      # TODO: Maybe in the future we should warn users if there are unused ParameterBuilders in a Profiler configuration.
+      - parameter_name: my_column_mean
+        class_name: MetricParameterBuilder
+        module_name: great_expectations.profiler.parameter_builder.metric_parameter_builder
+        metric_name: column.mean
+        metric_domain_kwargs: $domain.domain_kwargs
+      - parameter_name: my_column_min
+        class_name: MetricParameterBuilder
+        module_name: great_expectations.profiler.parameter_builder.metric_parameter_builder
+        metric_name: column.min
+        metric_domain_kwargs: $domain.domain_kwargs
+      - parameter_name: my_column_max
+        class_name: MetricParameterBuilder
+        module_name: great_expectations.profiler.parameter_builder.metric_parameter_builder
+        metric_name: column.max
+        metric_domain_kwargs: $domain.domain_kwargs
+
+      - parameter_name: false_positive_threshold_min
+        class_name: MultiBatchBootstrappedMetricDistributionParameterBuilder
+        module_name: great_expectations.profiler.parameter_builder.multi_batch_bootstrapped_metric_distribution_parameter_builder
+        # TODO: Not sure I understand the comment here about unioning - why would we want to add the batches from the call to profile? Wouldn't we expect that this parameter is calculated just from the explicitly defined batch_request below? (if this is useful, maybe there are two modes)
+        batch_request: # this should be unioned with the rest of the batch request from the call to profile
+          partition_request:
+            # NOTE: interpreted like python slices e.g. most recent 10 batches
+            partition_index: "-10:"
+        metric_configuration:
+          metric_name: column.min
+          # Process this metric on each of the domains in the list generated by the DomainBuilder:
+          metric_domain_kwargs: $domain.domain_kwargs
+        p_values:
+          min_value: $variables.false_positive_threshold
+          max_value: 1.0
+
+      - parameter_name: quantile_ranges
+        class_name: MultiBatchBootstrappedMetricDistributionParameterBuilder
+        module_name: great_expectations.profiler.parameter_builder.multi_batch_bootstrapped_metric_distribution_parameter_builder
```

Comment:
@anthonyburdi We can now shorten the module name to end at `great_expectations.profiler.parameter_builder` for the verbose, or omit it at all for the brief configuration YAML file.